pkg/alertmanager/alertmanager_http.go

```diff
@@ -23,31 +23,95 @@ var (
 			<p>{{ .Message }}</p>
 		</body>
 	</html>`))
+
+	statusTemplate = template.Must(template.New("statusPage").Funcs(map[string]interface{}{
+		"state": func(enabled bool) string {
```

Comment:
If I understand this right, I don't think we need this function.

Reply:
It was copied from previous implementation, but it's probably a leftover. Let's remove it.